src/themes/silveredit.css

```diff
@@ -0,0 +1,258 @@
+.slider {

```

Comment:
do we need to clone entire css from a theme ?

looks like its a copy of tailwind, could just import that

Reply:
Yeh better idea